dss/events/handlers/index.py

```diff
@@ -1,18 +1,22 @@
-
 """Lambda function for DSS indexing"""
 
 import json
 import os
 import re
+import uuid
 from urllib.parse import unquote
 
 import boto3
 import botocore
+import requests
+from elasticsearch.helpers import scan
 
-from ... import DSS_ELASTICSEARCH_INDEX_NAME, DSS_ELASTICSEARCH_DOC_TYPE
-from ...hcablobstore import BundleMetadata, BundleFileMetadata
-from ...util import create_blob_key
-from ...util.es import ElasticsearchClient
+from dss import (DSS_ELASTICSEARCH_INDEX_NAME, DSS_ELASTICSEARCH_DOC_TYPE,
+                 DSS_ELASTICSEARCH_QUERY_TYPE, DSS_ELASTICSEARCH_SUBSCRIPTION_INDEX_NAME,
+                 DSS_ELASTICSEARCH_SUBSCRIPTION_TYPE)
+from dss.util import create_blob_key
+from dss.hcablobstore import BundleMetadata, BundleFileMetadata
+from dss.util.es import ElasticsearchClient
```

Comment:
Please use relative imports when importing from the same package. Instead of `from dss`, use `from ...`.